xtask/src/ci.rs

```diff
@@ -68,7 +68,7 @@ impl CiTask {
         // 1. Make sure everything compiles
         cmd!("rustup run stable cargo check --workspace --all-features").run()?;
         cmd!("rustup run stable cargo check -p ruma-client --no-default-features").run()?;
-        cmd!("rustup run stable cargo check -p ruma-identifiers --no-default-features").run()?;
+        cmd!("rustup run stable cargo check -p ruma-common").run()?;
```

Comment:
Why remove `--no-default-features`?

Reply:
Won't it complain that `server` and `client` are disabled otherwise?